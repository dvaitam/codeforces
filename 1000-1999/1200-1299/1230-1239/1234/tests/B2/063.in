25 9
15 17 14 18 6 13 13 7 16 9 12 5 9 19 9 6 20 3 12 11 5 9 9 9 12
