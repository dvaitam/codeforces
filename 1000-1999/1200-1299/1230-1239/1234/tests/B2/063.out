9
11 3 20 19 5 12 9 16 7
